R/srcref_df.R

```diff
@@ -250,8 +250,8 @@ test_trace_mapping <- function(x) {
   if (!any(has_tests)) {
     empty_result <- matrix(
       integer(0L),
-      ncol = 4L,
-      dimnames = list(c(), c("test", "depth", "i", "trace"))
+      ncol = 5L,
+      dimnames = list(c(), c("test", "call", "depth", "i", "trace"))
     )
```

Comment:
This is unrelated to this PR, but is a necessary for use with the upcoming [PR to `covr`](https://github.com/dgkf/covr/tree/dgkf-min-dup-tests)

Reply:
Decided to address this most holistically, case-dependent on covr version.